src/components/two-factor-authentication/HardwareTokenWorkflow.vue

```diff
@@ -0,0 +1,149 @@
+<script setup lang="ts">
+  import { useDisplay } from 'vuetify';
+  import FormRow from '@/components/form/FormRow.vue';
+  import { ref, type Ref } from 'vue';
+  import { usePersonStore, type Personendatensatz, type PersonStore } from '@/stores/PersonStore';
+  import axios from 'axios';
+  import { useI18n, type Composer } from 'vue-i18n';
+
+  const { t }: Composer = useI18n({ useScope: 'global' });
+  const { mdAndDown }: { mdAndDown: Ref<boolean> } = useDisplay();
+  const dialogText: Ref<string> = ref('');
+  const proceeded: Ref<boolean> = ref(false);
+  const errorThrown: Ref<boolean> = ref(false);
+  const personStore: PersonStore = usePersonStore();
+  // eslint-disable-next-line @typescript-eslint/typedef
+  const emits = defineEmits(['updateHeader', 'onCloseClicked']);
+  emits('updateHeader', 'Hardware-Token zuordnen');
+
+  const serial: Ref<string> = ref('');
+  const otp: Ref<string> = ref('');
+
+  type Props = {
+    errorCode: string;
+    person: Personendatensatz;
+  };
+  const props: Props = defineProps<Props>();
+
+  function handleApiError(error: unknown): void {
+    errorThrown.value = true;
+    if (axios.isAxiosError(error)) {
+      if (error.response && error.response.data) {
+        const message: string = error.response.data.message || 'An unexpected error occurred.';
+        dialogText.value = message;
+      } else {
+        dialogText.value = 'An unexpected error occurred.';
```

Comment:
Die message kommt 1:1 so vom Backend und ist nicht konform zu unseren Translations. In anderen Responses bekommen wir einen Error Code zurück, den wir dann im Frontend übersetzen. Die message aus dem Backend sollte dem Frontend egal sein. Auch der Text aus dem else Zweig muss i18n-konform sein.

Reply:
done